tests/__fixtures__/exchanges/exchanges.fixtures.mts

```diff
@@ -107,6 +107,18 @@ export const responses = {
           },
         ],
       },
+      filters: [
+        {
+          criteria: ExchangeFilterLogicOperatorCriteria.ProductId,
```

Comment:
We need an alias for those types as well:

`ExchangeFilterLogicOperatorCriteria` ->  `ExchangeFilterConditionCriteria`
`ExchangeFilterLogicOperatorComparator` -> `ExchangeFilterConditionComparator`

Reply:
and done 🙂